src/CMakeLists.txt

```diff
@@ -166,9 +166,15 @@ if(Cython_FOUND)
           TARGETS ${name}
           LIBRARY DESTINATION lib
           )
-        INSTALL(
-          TARGETS ${name}
-          LIBRARY DESTINATION ${PYTHON_SITE_PACKAGES}
+        # Add symlinks to PYTHON_SITE_PACKAGES so python can find our cython modules
+        # Python will only import libraries with .so extensions
+        # so we hardcode the symlink with .so even on APPLE
+        INSTALL(CODE "
```

Comment:
Instead of using system links, I may suggest to use rpath, this may help to build python wheels more easier, since python sdist do not copy system links while making tar file.

Reply:
That's a good point.  The build is configured to use rpath for linking (and can confirm via `otool`), but I'm not sure that alone addresses the purpose of this symlink.  We install the cython modules (`pymodule`, `pyinfile`, and `eventhooks`) to `PYTHON_SITE_PACKAGES` because they are imported by the python interpreter and this is more robust than adding `INSTALL_PATH/lib` to the PYTHONPATH.  

I'm thinking it might be smart to just install the libraries twice - once as `.dylib` files to `/lib` and once as `.so` files to `PYTHON_SITE_PACKAGES` (because python will not import a shared library that is a `.dylib`). This is what was done prior to this PR, however it installed it as a `.dylib` both places.  It will likely need to be hardcoded as `.so` so the python interpreter can import on macOS.

I recognize that installing the same library twice is probably not great practice, so I'm open to suggestions to make this cleaner.